README.md

```diff
@@ -109,24 +109,158 @@ which appears to overlap with a subset of `OpenAssetIO`s concerns.
 
 ### System requirements
 
-- `linux` (`macOS` and `Windows` are currently unsupported)
-- `Python 3.7` or later
+Ubuntu Linux 20.04 is currently the only fully tested platform. Support
```

Comment:
[consider] Move the vagrant section to `Getting Started` and then move the full build instructions into `INSTALL`, and link from here?

Generally good to have easiest first, then increasing detail after to avoid Wall of Text syndrome. I appreciate that long-term this isn't what we'd want. But as we officially say Ubuntu is the only currently fully supported platform, that's prob OK for now?

Reply:
Q: Can you move it to the Wiki instead? with an optional link from the README.md to look at other options.

Reply:
As discussed offline, I've left this as-is. Also added a section on using the ASWF Docker image as part of 5ed9cdae

Reply:
Awesome, thanks. I am a little worried how long the `README.md` is now, what do you think about splitting the build stuff out into a separate doc (`INSTALL`)?

Reply:
What do you think about leaving the `ASWF docker` one on the main page, and linking to full build instructions- given the nature of our target audience?

Reply:
Good idea, will do.

Reply:
Split in 5212172

Reply:
Copy is perfect, how about this for a structural tweak (its a `git diff`)
[suggestion.txt](https://github.com/TheFoundryVisionmongers/OpenAssetIO/files/8048370/suggestion.txt)

Reply:
Applied with minor tweaks in 1d59523b